drivers/platform/aducm3029/aducm3029_i2c.c

```diff
@@ -126,8 +116,8 @@ static uint32_t set_transmission_configuration(struct i2c_desc *desc)
  * it is not used and must be set to NULL.
  * @return \ref SUCCESS in case of success, \ref FAILURE otherwise.
  */
-int32_t i2c_init(struct i2c_desc **desc,
-		 const struct i2c_init_param *param)
+int32_t aducm3029_i2c_init(struct i2c_desc **desc,
```

Comment:
make functions static. The should only be used through ops.

Reply:
nitpick. If function became static, rename is not needed, but this is not a big issue anyway

Reply:
Same, wanted to keep the code consistent. Function definitions are not included in the header file so they should be used only through ops.

Reply:
This means that they should be static :)